include/zephyr/net/net_pkt.h

```diff
@@ -234,6 +234,22 @@ struct net_pkt {
 	uint16_t vlan_tci;
 #endif /* CONFIG_NET_VLAN */
 
+	union {
+#if defined(CONFIG_NET_IPV4) && defined(CONFIG_NET_IPV4_FRAGMENT)
```

Comment:
Do we need the first part of the check here, wouldn't `#if defined(CONFIG_NET_IPV4_FRAGMENT)` be enough as the fragment symbol is not defined if IPv4 is not enabled.
Same comment for IPv6 check.

Reply:
Fixed